mitosheet/mitosheet/user/utils.py

```diff
@@ -32,6 +32,12 @@
 except ImportError:
     MITOSHEET_HELPER_ENTERPRISE = False
 
+try:
```

Comment:
I think there might be other checks of mitosheet private. I'll investigate now...